backend/src/plugins/fastify-roles/index.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const fp = require('fastify-plugin')
+const { getUserInfo } = require('../../helpers/auth')
+
+/**
+ * Fastify Roles plugin, that inserts user object into the request object for each api call.
+ *
+ * @param   {[type]}  fastify  [fastify description]
```

Comment:
I think the type here would be `FastifyInstance`. See: https://www.fastify.io/docs/latest/Reference/TypeScript/#fastifyfastifyinstance

Reply:
> I think the type here would be `FastifyInstance`. See: https://www.fastify.io/docs/latest/Reference/TypeScript/#fastifyfastifyinstance

You are probably right... I was kind of just leaving this, to see if it actually sticks around, otherwise seems like work we might or might not have to perform..